app/repositories/application-repository.js

```diff
@@ -19,15 +19,26 @@ async function getByEmail (email) {
     })
 }
 
-async function getAll (page = 0) {
-  return models.application.findAll(
-    {
-      order: [['createdAt', 'DESC']],
-      limit: 20,
-      offset: page
-    })
+async function getAll (limit, offset, sbi) {
+  const query = {
+    order: [['createdAt', 'DESC']],
+    limit: limit,
+    offset: offset
+  }
+  if (sbi) {
+    query.where = { 'data.organisation.sbi': sbi }
+  }
+  return models.application.findAll(query)
+}
+async function getApplicationCount (sbi) {
+  const query = {
+    order: [['createdAt', 'DESC']]
```

Comment:
Do you want to order the results when you are only interested in the number of them?